docs/user/README.md

```diff
@@ -35,7 +35,10 @@ You may also try removing any unused data left by the Docker engine - e.g.,
 After [creating a cluster](#creating-a-cluster), you can use [kubectl][kubectl]
 to interact with it by using the configuration file generated by `kind`:
 ```
-export KUBECONFIG="$(kind get kubeconfig-path)"
+Mac/Linux
+  export KUBECONFIG="$(kind get kubeconfig-path)"
+Windows Powershell
+ $env:KUBECONFIG="$(kind get kubeconfig-path)"
```

Comment:
powershell is not really required for the sake of using kind or setting env variables.

for the regular cmd shell we could do this:
```
for /f "delims=*" %i in ('kind get kubeconfig-path') do @set KUBECONFIG="%i"
```
which is far from pretty.

the alternative which gets my vote is:
```
kind get kubeconfig-path > kindpath && set /p KUBECONFIG=<kindpath && del kindpath
```